### DISCUSSION TITLE
web-app-mvc project doesn't work !

### DISCUSSION THREAD

**Discussion by dimalyshev** [NikiforovAll/keycloak-authorization-services-dotnet#139] [General]
Great contrib and  job!

...but notes in https://github.com/NikiforovAll/keycloak-authorization-services-dotnet/blob/main/docs/examples/web-app-mvc.md
make me wonder how to get working project (web-app-mvc) mentioned in notes?

Does anybody has successfull experience?
Thanx in advance...

**Comment by NikiforovAll** [comment]
Hi @dimalyshev,

Please take a look at #104. It contains the code to make it work, but it also highlights limitations on the OAuth2 flow that you can use.
